src/layout/_grid.scss

```diff
@@ -30,4 +30,8 @@
   &:nth-child(#{$col-num}n) {
     margin-right: 0;
   }
+
+  &:not(:nth-last-child(-n+#{$col-num})) {
```

Comment:
Giving vertical gutter (margin bottom) just to inner cells, not last ones, those within the last row.